test/index.js

```diff
@@ -1,12 +1,12 @@
-const fs = require('fs')
+const fs = require('fs-extra')
```

Comment:
`fs-extra` is missing from the `package.json`.
What about using [`cpy`](https://github.com/sindresorhus/cpy)? That library has no issues with dotfiles.